### DISCUSSION TITLE
Is Confererence working using asterisk confbridge in this project.

### DISCUSSION THREAD

**Discussion by deepaksharma9899** [InnovateAsterisk/Browser-Phone#460] [General]
I am so thankful to you for this project.
Can you please tell me if conference is working in this project.

**Comment by InnovateAsterisk** [comment]
Yes, the Asterisk ConfBridge module is mostly supported. Audio and video works (in all modes), notifications works, the only thing that’s not working is the participants text messaging.

**Reply by deepaksharma9899** [reply]
Thanks for replying. I tried running ConfBridge module. I am getting audio from all the participants, so audio part is working properly. But in case of video it is showing black screen in the conference and showing only one incoming screen window despite of many participants. How to resolve it? Thanks in advance.

**Comment by InnovateAsterisk** [comment]
This may be the way you have configured ConfBridge. 

> video_mode
Sets how confbridge handles video distribution to the conference participants. Note that participants wanting to view and be the source of a video feed MUST be sharing the same video codec. Also, using video in conjunction with with the jitterbuffer currently results in the audio being slightly out of sync with the video. This is a result of the jitterbuffer only working on the audio stream. It is recommended to disable the jitterbuffer when video is used.
none - No video sources are set by default in the conference. It is still possible for a user to be set as a video source via AMI or DTMF action at any time.
follow_talker - The video feed will follow whoever is talking and providing video.
last_marked - The last marked user to join the conference with video capabilities will be the single source of video distributed to all participants. If multiple marked users are capable of video, the last one to join is always the source, when that user leaves it goes to the one who joined before them.
first_marked - The first marked user to join the conference with video capabilities is the single source of video distribution among all participants. If that user leaves, the marked user to join after them becomes the source.
sfu - Selective Forwarding Unit - Sets multi-stream operation for a multi-party video conference.

SFU is the mode that shows all participants all the time, like Meet, Zoom, Teams, etc

**Reply by InnovateAsterisk** [reply]
https://wiki.asterisk.org/wiki/display/AST/Asterisk+16+Configuration_app_confbridge

**Reply by deepaksharma9899** [reply]
Thankyou for your reply.
Finally conference is running on my server.
Now there is a problem, my video is running for only 2 minutes and I am getting this warning repeatedly
"res_srtp.c:508 ast_srtp_protect: SRTP protect: replay check failed (index too old)"
How to solve this? Thanks 

Also, I think the video frames in the conference should have the identity of the caller not of the conference room they are dialing.
